Signal/src/call/CallService.swift

```diff
@@ -1434,7 +1435,7 @@ protocol CallServiceObserver: class {
         AssertIsOnMainThread()
 
         if case CallError.assertionError(description:let description) = error {
-            owsFail(description)
```

Comment:
I don't think we should get rid of this. Is there some assertion that you're regularly seeing which is making this inconvenient? Maybe we should revisit that assertion.

Reply:
I hit this assert while hanging up a test-device-to-simulator call.  I assumed it was yet another over-aggressive assert in `CallService`.  I'll restore it, but I'm not sure we should `owsFail` edge cases which will recover gracefully in production.